swarm/index.md

```diff
@@ -4,6 +4,13 @@ keywords: docker, swarm,  clustering
 title: Docker Swarm
 ---
 
+This section refers to standalone Docker Swarm. If you use Docker 1.12 or higher,
+[Swarm mode](../engine/swarm/index.md) is integrated with Docker Engine, and you
+should use Swarm mode instead.
```

Comment:
@aaronlehmann I'm fine with your wording.  I think our position is that 1.12 services is recommended over `docker/swarm`

@mstanleyjones The older `docker/swarm` is simpler so it's better for some setups like Jenkins slaves.  But it's not a specific enumerable set of features, it's more about a simpler model.